svensk-exegetisk-arsbok.csl

```diff
@@ -0,0 +1,782 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="chicago" demote-non-dropping-particle="sort-only" default-locale="sv-SE">
```

Comment:
at least in Zotero, you can't override the style default. In Zotero 4.27+ you should be able to adjust the language used by any style without a default locale in the document preferences, though.